### PR TITLE
feat(doc) update retention/replication doc for 2.0

### PR DESCRIPTION
**Pull request by bitsf**
update retention and replication doc for harbor 2.0

### CODE REVIEW

File: docs/administration/configuring-replication/create-replication-rules.md [hunk 1]
```diff
@@ -48,6 +48,10 @@ A replication endpoint must exist before you create a replication rule. To creat
    If you do not enter a namespace, resources are placed in the same namespace as in the source registry. 
 
    ![Destination and namespaces](../../../img/replication-rule4.png)
+   
+   **NOTE:** Because there's some API change in 2.0 release in order to support [OCI](https://github.com/opencontainers/distribution-spec).
+   You **can not** replication from harbor 1.x to 2.0, and you **can not** replication artifacts with **manifest list** from 2.0 to 1.x.
```
Comment (renmaosheng) [inline docs/administration/configuring-replication/create-replication-rules.md:53]: could we replicate image from 2.0 to 1.x? if not, I don't think we need the second sentence. just say : "You **can not**  replicate artifacts between harbor 1.x and Harbor 2.0".

File: docs/working-with-projects/working-with-images/create-tag-retention-rules.md [hunk 2]
```diff
@@ -94,6 +104,8 @@ In this example, the rules are applied to the following 7 tags:
 - `3.1-your_repo-prod`
 - `4.4-your_repo-prod`
 
+Because there's no untagged artifacts, so no difference whether check the checkbox.
```
Comment (xaleeks) [inline docs/working-with-projects/working-with-images/create-tag-retention-rules.md:107]: ‘there are no untagged artifacts'

File: docs/administration/configuring-replication/create-replication-rules.md (hunk 1 above)
Comment (bitsf) [reply docs/administration/configuring-replication/create-replication-rules.md:53]: 我们可以从2.0往1.x复制非manifast list类型的

Comment (xaleeks) [reply docs/administration/configuring-replication/create-replication-rules.md:53]: it should also say 'you can not replicate' instead of 'replication'

Comment (xaleeks) [reply docs/administration/configuring-replication/create-replication-rules.md:53]: agree with Steven, just say you cannot replicate btw ...

File: docs/working-with-projects/working-with-images/create-tag-retention-rules.md [hunk 3]
```diff
@@ -15,58 +15,68 @@ A tag retention rule has 3 filters that are applied sequentially, as described i
...
-In this case, all of the images in repository A are retained because they have been pulled in the last 7 days. None of the images in repositories B to E are retained, because none of them has been pulled in the last week. In this example, 100 images are retained, as opposed to 34 images in example 1.
+In this case, all of the 100 tagged artifacts in repository A are retained because they have been pulled in the last 7 days. None of the images in repositories B to E are retained, because none of them has been pulled in the last week. In this example, 100 images are retained, as opposed to 34 images in example 1.
+And all untagged artifacts are deleted.
+
+### Example 3
+
+This example uses the same project and repositories as example 2, but checked "untagged artifacts", then all artifacts in repository A are retained.
+
+In this case, all of the 103 artifacts in repository A are retained because they have been pulled in the last 7 days. None of the images in repositories B to E are retained, because none of them has been pulled in the last week. In this example, 103 artifacts are retained, as opposed to 100 artifacts in example 2.
+And all untagged artifacts are deleted.
```
Comment (renmaosheng) [inline docs/working-with-projects/working-with-images/create-tag-retention-rules.md:47]: should be maintained? thanks.

**Pull request merged**